tests/cypress/support/commands.ts

```diff
@@ -709,3 +715,108 @@ Cypress.Commands.add('compareClusterResourceCount', (clusterName) => {
     })
   })
 })
+
+Cypress.Commands.add('createNewFleetWorkspace', (newWorkspaceName) => {
+  // Create new workspace
+  cy.accesMenuSelection('Continuous Delivery', 'Advanced', 'Workspaces');
+  cy.clickButton('Create')
+  cy.contains('Workspace:').should('be.visible');
+  cy.typeValue('Name', newWorkspaceName);
+  cy.clickButton('Create');
+  cy.filterInSearchBox(newWorkspaceName)
+  cy.verifyTableRow(0, 'Active', newWorkspaceName);
+})
+
+Cypress.Commands.add('enableFeatureFlag', (flagName) => {
+  // Enable given Feature Flag for the cluster.
+  cy.accesMenuSelection("Global Settings", "Feature Flags");
+  cy.contains('.title', 'Feature Flags').should('be.visible');
+
+  cy.filterInSearchBox(flagName);
+  cy.verifyTableRow(0, 'Disabled', flagName);
+  cy.open3dotsMenu(flagName, 'Activate');
+
+  // Activate the Feature Flag
+  cy.get('div.card-title')
+  .should('be.visible')
+  .then(($el) => {
+    if ($el.text().includes('Are you sure?')) {
+      // Check that 'Are you sure?' is visible before activating.
+      cy.checkModalCardTitle('Are you sure?', true, false);
+      cy.clickButton("Activate");
+    }
+  });
+
+  // Check that 'Are you sure?' is not visible anymore.
+  cy.checkModalCardTitle('Are you sure?', false, false);
+
+  // After activation, Rancher pod will restart.
+  // Wait for 'Waiting for Restart' to disappear.
+  cy.checkModalCardTitle('Waiting for Restart', true)
+})
+
+Cypress.Commands.add('checkModalCardTitle', (expectedText, shouldHaveText, waitForRestart=true) => {
```

Comment:
It would be nice to set most common behavior here (either false or true) as for `waitForRestart` argument

Reply:
By default, we have to wait for restart on some featureFlags. So, I defaulted `true` which says that restart the cluster.
If we don't want to restart then simply pass the Param: false.

If you want to me to reverse it, i.e. pass `false` by default and when checking wait for restart pass `true`?

Reply:
You defaulted to `true` the `waitForRestart`argument. What I meant is to do the same for the `shouldHaveText` argument. I'd give it a default value (either `true` or `false`).

Reply:
Yes, I will do it :+1: